src/legacyIpfs3boxAccessController.js

```diff
@@ -34,11 +34,18 @@ class LegacyIPFS3BoxAccessController {
     const publicKey = entry.v === 0 ? entry.key : await publicKeyFromDID(entry.identity.id)
     if (this.write.includes(publicKey) ||
       this.write.includes('*')) {
-      return true
+      return entry.v === 0 ? true : await identityProvider.verifyIdentity(entry.identity)
     }
     return false
   }
 
+
+  const publicKey = entry.v === 0 ? entry.key : entry.identity.id
+ if (this.write.includes(publicKey) ||	    if (this.write.includes(publicKey) || this.write.includes('*')) {
+   this.write.includes('*')) {	      return entry.v === 1 ? await identityProvider.verifyIdentity(entry.identity) : true
+   return true
+ }	    }
```

Comment:
This code shouldn't be here?

Reply:
gosh no, was referencing code from my orbitd pr, just removed

Reply:
xD